modules/wfa-qt/CMakeLists.txt

```diff
@@ -19,6 +19,13 @@ zephyr_library_compile_options(
 )
 
 zephyr_library_compile_definitions(CONFIG_CTRL_IFACE_ZEPHYR)
+zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_P2P
+  CONFIG_P2P
```

Comment:
kconfig settings should not be applied using compile definitions but through the correct Kconfig symbol being enabled.

Please explain ?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/25841/files#r2568033894